Automattic/wp-calypso
# Marketplace: enables feature in staging.

#### Changes proposed in this Pull Request

* Enable marketplace testing in staging environment.

#### Testing instructions



* Click the Calypso Live link
* Go to `/marketplace/test/[DOMAIN]` 
* You should be presented a developer screen with relevant actions





Related to pdh6GB-4-p2
